website/tutorial/part_2/part_2a/04_html_server.md

```diff
@@ -6,66 +6,68 @@
   </tr>
 </table>
 
-
 # The HTML Server Service
 
-Let's build the first service for our application:
-the HTML server!
-If we were building our Todo app as a traditional monolith,
-this would be the only code base
-and perform all of the application's functionality:
-receiving requests,
-storing todo items, user accounts, and API tokens in the database,
-configuring and using the search engine,
-tracking user sessions and permissions,
-rendering HTML for the browser, JSON for the REST API,
-sending out emails,
-etc.
-
-In Exosphere's microservice world,
-each code base has only one responsibility.
-The html server's job is to interact with the user via an HTML UI.
-Most of the things mentioned above are not a direct part of this responsibility,
-and are therefore implemented outside of the html server,
-as separate services.
-Because of this much narrower set of responsibilities,
-the html server is a lot smaller and simpler
-than it would be in a traditional monolithic application.
-
-Since our html server is so simple,
-we'll build it using [ExpressJS](http://expressjs.com).
-Exosphere provides a template for building ExpressJS html servers.
-Let's use it:
+Let's build the first service for our application: the HTML server! If we were
```

Comment:
Please try to match the existing flow of line breaks. The existing text breaks at logical breaking points like commas, subclauses, and each time a thought chain ends. This has several advantages: it makes it much easier to read the markdown code (okay, that's debatable, but I find it easier), and it limits changes to the respective line their are in. If we followed a constant line length, then just adding one world somewhere potentially changes all lines after it, causing massive diffs where it is hard to spot the actual change. Plus, not everybody has a sophisticated editor that automatically breaks text at constant line lengths, and we don't want to exclude these people. Thanks!  :)

Reply:
I agree. 

I think my editor took this page by storm. I caught it in it's antics in the the following page and disabled that behavior. I'll revert all of it's formatting changes in this page.